edge/services/helloMMS/README.md

```diff
@@ -47,60 +47,59 @@ hzn exchange node confirm
 
```

Comment:
I know this isn't part of what you edited but if you wanted to update step 3 of the Preconditions to be similar to `cpu2evtstreams` and `helloworld` and add the new step 6 to Preconditions as well:

3. As part of the infrasctucture installation process for IBM Edge Computing Manager a file called `agent-install.cfg` was created that contains the values for `HZN_ORG_ID` and the exchange and css url values. Locate this file and set those environment variables in your shell now:
```bash
eval export $(cat agent-install.cfg)
```
...
6. If you have not done so already, unregister your node before moving on:
```bash
hzn unregister -f
```

Reply:
I was wondering about this. Thx for this info! I added it.